modules/sql-engine/src/integrationTest/sql/subquery/any_all/test_correlated_any_all.test

```diff
@@ -1,7 +1,7 @@
 # name: test/sql/subquery/any_all/test_correlated_any_all.test
 # description: Test correlated ANY/ALL subqueries
 # group: [any_all]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-19401
+# Ignore https://issues.apache.org/jira/browse/IGNITE-22260
```

Comment:
We agreed to close the ticket. our current behavior we consider as correct

Reply:
got it